src/layouts/partials.tsx

```diff
@@ -203,6 +203,14 @@ export function Footer({ authenticated }: IFooterProperties): ReactElement {
                   System status
                 </a>
               </li>
+              <li className="govuk-footer__list-item">
+                <a
+                  className="govuk-footer__link"
+                  href="/performance"
```

Comment:
Is this correct? You should be able to access /performance-london or /performance-ireland wherever you are